Makefile

```diff
@@ -0,0 +1,15 @@
+.PHONY: all dev deps
+
+GO ?= go
+HUGO ?= hugo
+
+HUGO_DEV_FLAGS ?= --bind 0.0.0.0
+
+all:
+	cd src && $(HUGO)
+
+dev:
+	cd src && $(HUGO) server $(HUGO_DEV_FLAGS)
+
+deps:
+	git submodule update --init --remote
```

Comment:
there are no git submodules at least for now

Reply:
@karasz https://github.com/jpi-oss/jpi-oss-website/pull/4 adds one :stuck_out_tongue_closed_eyes: